src/server.cc

```diff
@@ -34,6 +34,7 @@
 #ifdef HAVE_CURL
 #include <curl/curl.h>
 #include <iostream>
+#include <upnptools.h>
```

Comment:
This should go outside #ifdef

Reply:
Yep, it needs its own #ifdef! Not sure why editor decided to add it there :grin: